ow_power_system/config/system.cfg

```diff
@@ -28,16 +28,21 @@ voltage_range: 0.1			        # volts
 # values from other planetary rovers, and some are just guesses.
 
 # Baseline values (watts) for constant power draws.
-baseline_power_camera: 0.1
-baseline_power_communications: 0.6
 baseline_power_computing: 0.1
 baseline_power_heating: 1.2
 baseline_power_science_instr: 0.6
 baseline_power_sample_handling: 0.6
 baseline_power_other: 2.2
 
+
 # Power coefficients for dynamically computed power draw.
-wattage_coefficient_lights: 0.3    # power draw (W) per light at max intensity
+power_lights_coefficient: 0.3       # power draw (W) per light at max intensity
+power_camera_controller: 1.0        # power draw of the camera controller
```

Comment:
Isn't this a baseline draw, hence better named `baseline_power_camera_controller` ?

Reply:
I think I'd like to prefix any power draw related value with `power_`, so how about we do `power_baseline_camera_controller`?

Addressing your other comment, what if we made `power_lights_coefficient` into `power_active_lights`?

Reply:
done